tests/test_prompts.py

```diff
@@ -1042,7 +1042,7 @@ def test_judge_check_contamination():
 
 
 def test_generic_lean4_prompt():
-    prompt = get_prompt('generic/lean4', 'deepseek-prover')
+    prompt = get_prompt('lean4/lean4', 'deepseek-prover')
```

Comment:
lean4/lean4 is not very descriptive, maybe use something like lean4/prove instead?

Reply:
Done